pkg/status/status_collector.go

```diff
@@ -48,7 +49,8 @@ type statusCollector struct {
 
 	allProbesInitialized bool
 
-	statusParams statusParams
+	statusParams    statusParams
+	bgpStatusGetter healthcheck.BGPStatusGetter
```

Comment:
nit: Please add this to field `statusParams` instead (it still will be injected via Hive)